### DISCUSSION TITLE
#48 - Кирилл Грищук - Java собеседования, Литкод и полезный лайвкодинг

### DISCUSSION THREAD

**Discussion by giscus** [javaswag/javaswag.github.io#8] [Announcements]
# #48 - Кирилл Грищук - Java собеседования, Литкод и полезный лайвкодинг

В 48 выпуске подкаста Javaswag поговорили с Кириллом Грищуком о Java собеседованиях, Литкоди и полезном лайвкодинге

https://javaswag.github.io/episode/48/

**Comment by romanvolykhin** [comment]
Крутой выпуск. Замотивировало решать литкод
